src/entities/currency.ts

```diff
@@ -3,6 +3,7 @@ import JSBI from 'jsbi'
 import { ChainId, SolidityType } from '../constants'
 import { validateSolidityTypeInstance } from '../utils'
 
+const BURN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
There's no burn address, we need a better name :)

Reply:
Changed to NULL_ADDRESS